vuepress/docs/next/tutorials/compose/content-types-tutorial.md

```diff
@@ -1,101 +1,70 @@
 
-A Content Type is used to represent or to model a Content.
-It is characterized by a set of [Attribute Types](./content-attributes.md), each of which defines the value of their properties, called Attributes. A Content is
-an instance or specialization of the Content Type.
+A Content Type represents or models a Content. Alternatively, a Content is an instance or specialization of a Content Type. 
 
-## Create a Content Type
-
-Creating Contents Types is fairly simple. The Content Types are managed
-through the Web CMS, its user interface is made available in the App
-Builder application.
-
--   Go to `Content → Types`
+Content Types are characterized by [Attribute Types](./content-attributes.md). Each Attribute Type consists of one or more properties, known as Attributes. An Attribute is defined by its Attribute Type.
 
--   Click on the **Add Content Type** button to create a new Content
-    Type.
-
-![image](./img/ContentTypes2.png)
+## Create a Content Type
 
--   You be asked to provide
+Content Types are managed via the Web CMS, through a user interface provided by the Entando App Builder. Follow the steps below to create and configure a Content Type.
 
-    -   **Code**. The code is the unique identifier of the content type,
-        that must be 3 uppercase letters, (e.g. CNN). It’s mandatory.
+**1. From the left menu of the App Builder, go to `Content` → `Types`.**
 
-    -   **Name.** The name must be max 50 characters uppercase and
-        lowercase letters, including numbers and special characters,
-        (e.g. Conference\_News). It’s mandatory.
+**2. Click the `Add` button in the upper right corner.**
 
-![image](./img/ContentTypes3.png)
+![addContentType](./img/content_types1.png)
 
--   Select the **Save** button, then you’ll see
+**3. Configure the Content Type.**
 
-    -   **Content Type Information**:
+When adding a Content Type it is mandatory to enter `Code` and `Name` values consistent with the following parameter definitions:
 
-        -   **Code**. The value is pre-filled.
+- `Code`: A unique identifier of the Content Type that must be 3 uppercase letters (e.g. CNN).
 
-        -   **Name**. The name is pre-filled.
+- `Name`: A string value (50 characters max) consisting of uppercase and lowercase letters, numbers and/or special characters (e.g. Conference\_News).
 
-    -   **Metadata**
+Field selections in the Metadata and Attributes sections are not mandatory, but to specify a meaningful Content Type you must choose the appropriate `Type` from the Attributes drop-down (e.g. 'Text'). The available Attribute Types define Attributes to characterize the Content.
 
-        -   **Page for on-the-fly publishing**
+![configureContentType](./img/content_types2.png)
 
-        -   **Default content template**
+**4. Click the `Add` button.**
 
-        -   **Default content template for lists**
+This initializes the configuration of the selected Attribute Type. Each Attribute Type requires its own configuration. 
```

Comment:
wondering if 'initializes' is the correct word here? reading this, just seemed like these steps just sets up the defining attribute types that define content whereas initializing involves something more complex.

Reply:
hm i think you're right about implied complexity. i think "launches" is a better choice.